### PR TITLE
feat(documentation):new folder created

### PR DESCRIPTION
**Pull request by Unsharm**
## Description
In this pull request, we have introduced a new folder named "documentation". Additionally, we have established a link from the 'kit creation' button directly to this folder.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by SujitMBRDI — DISMISSED** [review]
LGTM

File: documentation/kit-artifacts.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+**Quick Setup Guide / Installation Instructions**
+
+- Prerequisites (if they exist)
+- Authorization information
+- Deployment
+  **Deployment Scripts**
+
+## Architecture View
+
+The architecture view is an optional chapter. It offers an architecture perspective including the main building blocks. It can be extended with more architecture documentation like an ARC42.
```
Comment (mhellmeier) [inline documentation/kit-artifacts.md:209]: If a KIT will provide the architecture view, it should contain two chapters:

- Building Block View (for the main components)
- Runtime View (for the interaction between these components)

This structure is in line with arc42.

File: documentation/kit-artifacts.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+- SPDX-License-Identifier: CC-BY-4.0
+- SPDX-FileCopyrightText: 2023,2023 ZF Friedrichshafen AG
+- SPDX-FileCopyrightText: 2023,2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+- SPDX-FileCopyrightText: 2023,2023 SAP SE
+- SPDX-FileCopyrightText: 2023,2023 Volkswagen AG
+- SPDX-FileCopyrightText: 2023,2023 Robert Bosch GmbH
+- SPDX-FileCopyrightText: 2023,2023 Mercedes Benz Group
+- SPDX-FileCopyrightText: 2023,2023 BASF SE
+- SPDX-FileCopyrightText: 2023,2023 Schaeffler AG
+- SPDX-FileCopyrightText: 2023,2023 Contributors to the Eclipse Foundation
```
Comment (mhellmeier) [inline documentation/kit-artifacts.md:155]: `2023,2023` is the wrong format and not in line with the Eclipse Foundation Project Handbook: https://www.eclipse.org/projects/handbook/#ip-copyright-headers

**Review by mhellmeier — CHANGES_REQUESTED** [review]
Other comments are mentioned in #819

File: documentation/kit-artifacts.md (hunk 2 above)
Comment (maximilianong) [reply documentation/kit-artifacts.md:155]: @Unsharm please correct the example. If you have questions, feel free to contact @SujitMBRDI or me directly.

File: documentation/kit-artifacts.md (hunk 1 above)
Comment (Unsharm) [reply documentation/kit-artifacts.md:209]: > @Unsharm can you add this to the content, thanks.

>>Hi @maximilianong, okay  i am doing the changes.

**Review by manojava-gk — APPROVED** [review]
lgtm

**Review by maximilianong — APPROVED** [review]

**Review by SujitMBRDI — APPROVED** [review]

**Comment by maximilianong** [comment]
@mhellmeier can you please check.

**Pull request merged**